packages/radio/src/react/__specs__/index.spec.tsx

```diff
@@ -29,7 +29,7 @@ describe('Radio', () => {
       </Radio.Group>
     )
     const radio = container.querySelector('input')
```

Comment:
```suggestion
    const radio = container.querySelector('input')!
```

Reply:
don't ask me why...but you're supposed to do the bang on this line instead of the next